components/Checkbox/Checkbox.jsx

```diff
@@ -108,31 +108,40 @@ const HiddenCheckbox = styled.input.attrs({
 
 HiddenCheckbox.displayName = 'HiddenCheckbox';
 
-const Checkbox = ({ label, error, id, ...rest }) => (
+const Checkbox = ({ children, error, id, label, value, ...rest }) => (
   <Wrapper>
     <CheckboxWrapper>
-      <HiddenCheckbox id={id} error={error} {...rest} />
+      <HiddenCheckbox id={id} error={error} value={value} {...rest} />
       <CheckIcon />
-      <CheckboxLabel htmlFor={id}>{label}</CheckboxLabel>
+      <CheckboxLabel htmlFor={id}>{children || label || value}</CheckboxLabel>
     </CheckboxWrapper>
-    {error && <ErrorMessage>{error}</ErrorMessage>}
+    {error && typeof error === 'string' && <ErrorMessage>{error}</ErrorMessage>}
   </Wrapper>
 );
 
 Checkbox.defaultProps = {
-  checked: undefined,
+  checked: false,
+  children: '',
   disabled: false,
   error: '',
   id: '',
   label: '',
+  onChange: () => {},
+  value: '',
 };
 
 Checkbox.propTypes = {
+  children: PropTypes.string,
```

Comment:
Please, consider sorting `PropTypes.*` following this pattern:
 - Boolean
 - String
 - Others (fund, unions, etc.)

It will render `AutoProps` properly :nail_care:

Reply:
Done ✅